src/Explorer/Notebook/NotebookUtil.ts

```diff
@@ -328,4 +329,21 @@ export class NotebookUtil {
     link.click();
     document.body.removeChild(link);
   }
+  public static getDownloadModalConent(fileName: string): string {
+    if (this.isPhoenixEnabled()) {
+      return `In order to download and run your notebook, a local temporary environment will be created for your account to view, edit and run the notebooks for a dedicated period of time. If you need the notebook to be stored permanently, please connect to github and then download the notebook to your github repo.`;
```

Comment:
> github repo. [](http://example.com/codeflow?start=303&length=13)

can we add a link to our github documentation?

@deborahc ?

Reply:
@deborahc do have any update on this. Waiting on it.